aspnetcore/blazor/tooling.md

```diff
@@ -1,27 +1,39 @@
 ---
 title: Tooling for ASP.NET Core Blazor
 author: guardrex
-description: Learn about the tools available to build Blazor apps on various platforms.
+description: Learn about the tools available to build Blazor apps and how to use them.
 monikerRange: '>= aspnetcore-3.1'
 ms.author: riande
-ms.custom: mvc, linux-related-content
-ms.date: 02/09/2024
+ms.custom: mvc
+ms.date: 06/13/2024
 uid: blazor/tooling
-zone_pivot_groups: operating-systems
+zone_pivot_groups: tooling
 ---
 # Tooling for ASP.NET Core Blazor
 
 [!INCLUDE[](~/includes/not-latest-version.md)]
 
-This article describes tools for building Blazor apps on various platforms. Select your platform at the top of this article.
+This article describes tools for building Blazor apps using several tools:
 
-:::zone pivot="windows"
+* [Visual Studio (VS)](https://visualstudio.microsoft.com): The most comprehensive integrated development environment (IDE) for .NET developers on Windows. Includes an array of tools and features to elevate and enhance every stage of software development.
+* [Visual Studio Code (VS Code)](https://code.visualstudio.com) is an open source, cross-platform IDE that can be used to develop Blazor apps.
+* [.NET CLI](/dotnet/core/tools/): The .NET command-line interface (CLI) is a cross-platform toolchain for developing, building, running, and publishing .NET applications. The .NET CLI is included with the [.NET SDK](/dotnet/core/sdk) and runs on any platform supported by the SDK.
 
-To create a Blazor app on Windows, use the following guidance:
+For assistance choosing between VS and VS Code, see the following resources:
+
+* [What is Visual Studio?](/visualstudio/get-started/visual-studio-ide)
+* [Visual Studio Code: Getting Started](https://code.visualstudio.com/docs)
+* [Wondering which tool is best for you? We can help](https://visualstudio.microsoft.com/downloads/#picker)
```

Comment:
I think the descriptions of the tooling options above is already good enough to pick the preferred option, so I think we can just remove these links.

```suggestion
```